### PR TITLE
Some additional formatting, remove out of date sections from README

### PR DESCRIPTION
**Pull request by grandizzy**

### CODE REVIEW

File: src/ERC20Pool.sol [hunk 1]
```diff
@@ -108,7 +108,7 @@ contract ERC20Pool is IPool, Clone, Interest {
         }
 
         lpBalance[recipient_][price_] += lpTokens;  // update lender lp balance for current price bucket
-        totalQuoteToken               += amount_;   // update quote token accumulator
+        totalQuoteToken                 += amount_;  // update quote token accumulator
```
Comment (EdNoepel) [inline src/ERC20Pool.sol]: GitHub shows this as properly aligned in the old commit, now misaligned in the new one.

File: src/libraries/Buckets.sol [hunk 2]
```diff
@@ -389,9 +386,9 @@ library Buckets {
             if (amount_ > curLupDebt) {
                 bucket_.debt      += curLupDebt;
                 bucket_.onDeposit -= curLupDebt;
-                curLup.debt        = 0;
+                curLup.debt       = 0;
                 curLup.onDeposit  += curLupDebt;
-                amount_           -= curLupDebt;
+                amount_          -= curLupDebt;
```
Comment (EdNoepel) [inline src/libraries/Buckets.sol]: Not sure if we're aligning operators, RHS variables, or both here.  But line 391 seems it was spaced properly in the old commit.

**Review by EdNoepel — COMMENTED** [review]
Two spacing issues.

File: src/ERC20Pool.sol (hunk 1 above)
Comment (grandizzy) [reply src/ERC20Pool.sol]: yeah, I reverted, weird thing, VSC shows not aligned but whnd I open same file in vi it looks good :man_shrugging:

File: src/libraries/Buckets.sol (hunk 2 above)
Comment (grandizzy) [reply src/libraries/Buckets.sol]: same as above, I reverted it

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Pull request merged**